Client/Application/WebServer.swift

```diff
@@ -34,14 +34,37 @@ class WebServer {
     }
 
     @discardableResult func start() throws -> Bool {
-        if !server.isRunning {
-            try server.start(options: [
-                GCDWebServerOption_Port: AppInfo.webserverPort,
-                GCDWebServerOption_BindToLocalhost: true,
-                GCDWebServerOption_AutomaticallySuspendInBackground: false, // done by the app in AppDelegate
-                GCDWebServerOption_AuthenticationMethod: GCDWebServerAuthenticationMethod_Basic,
-                GCDWebServerOption_AuthenticationAccounts: [sessionToken: ""]
-            ])
+        if server.isRunning {
+            return true
+        }
```

Comment:
nit: would be nice to guard it

Reply:
Not sure a guard adds anything here, except for 9 extra characters in the source file.

Reply:
Guard makes it implicit you're not going further into that function, without even looking as to what there is inside the if.

Reply:
OK, I've replaced it with a guard. (BTW -  the original code wrapped the entire function in an `if !isRunning` {}